fetch.bs

```diff
@@ -5744,6 +5751,14 @@ optional boolean <var>forceNewConnection</var> (default false), run these steps:
          <li><p>If <var>request</var>'s <a for=request>mode</a> is "<code>websocket</code>" and
          <var>status</var> is 101, then <a for=iteration>break</a>.
 
+         <li><p>If <var>timingInfo</var>'s
+         <a for="fetch timing info">first interim network-response start time</a> is 0, then set
+         <var>timingInfo</var>'s
+         <a for="fetch timing info">first interim network-response start time</a> to
+         <var>timingInfo</var>'s <a for="fetch timing info">final network-response start time</a>.
+
+         <li><p>If <var>status</var> is 101, <a for=iteration>break</a>.
```

Comment:
Where is this coming from? This seems wrong.

Reply:
Removed